geode-membership/src/main/java/org/apache/geode/distributed/internal/membership/gms/fd/GMSHealthMonitor.java

```diff
@@ -152,7 +152,7 @@
   /**
    * Members undergoing final checks
    */
-  private final List<ID> membersInFinalCheck =
+  final List<ID> membersInFinalCheck =
```

Comment:
do we need to add `@VisibleForTesting` annotation here?

Reply:
yes